.github/workflows/branch-deploy.yaml

```diff
@@ -175,7 +175,7 @@ jobs:
       needs.validate-user.outputs.is_org_member == 'true' &&
       github.event_name == 'workflow_dispatch' &&
       (always() && needs.publish_nr_launcher.result == 'success')
-    uses: flowfuse/github-actions-workflows/.github/workflows/build_container_image.yml@v0.38.0
+    uses: flowfuse/github-actions-workflows/.github/workflows/build_container_image.yml@v0.39.0
```

Comment:
Only out of curiosity.

Is there a reason some of these are single quoted and this one isn't?